wasp/src/main/java/com/orhanobut/wasp/VolleyImageHandler.java

```diff
@@ -130,6 +132,10 @@ public void run() {
 
         @Override
         public void onErrorResponse(VolleyError error) {
+            String message = error.getMessage();
```

Comment:
is there any case that error is null? I think would be nice to make a null check here